lepton/image.go

```diff
@@ -74,23 +74,36 @@ func addHostName(m *Manifest, c *Config) {
 }
 
 func addFilesFromPackage(packagepath string, m *Manifest) {
```

Comment:
I don't think these changes work, i might be mistaken. Also there is no point in traversing the directory twice.

Reply:
I tested these changes and they are working. Also I'm not traversing directory twice: first `filepath.Walk` traversing "sysroot", second one is it's parent directory ignoring "sysroot".